src/background.js

```diff
@@ -11,17 +11,21 @@ import {
 import { getGroupProviders, getProviderTargetURL } from "./js/shared/misc";
 import ConfigFile from "./js/shared/config_file";
 import LocalStore from "./js/shared/local_store";
+import Analytics from "./js/shared/analytics";
 
 // Install handler.
 chrome.runtime.onInstalled.addListener(async function (details) {
   // Check if migration script should be run.
   var previous = _.get(details, "previousVersion");
+  var properties = { "Current Version": chrome.runtime.getManifest().version };
+
   if (!_.isEmpty(previous) && previous.split(".")[0] === "4") {
     // Open migration screen.
     chrome.tabs.create({
       url: "migration.html?previous=" + previous,
       selected: true,
     });
+    properties["Previous Version"] = previous;
```

Comment:
This will set `Previous Version` only when it's version 4, and it should be always added.  

I think you can just add it when declaring `properties` (e.g. `var properties = { "Current Version": ..., "Previous Version": previous };`